nautobot/extras/views.py

```diff
@@ -1223,6 +1226,19 @@ def post(self, request, class_path=None, pk=None):
                     **job_model.job_class.serialize_data(job_kwargs),
                 )
 
+                return_url = request.POST.get("_return_url")
```

Comment:
There's a change in behavior here that we'll want to make sure is desired. Previously job buttons ignored any of the error cases above this `else` block and were always redirected back to the object detail page.

Reply:
Good catch. Let me see if there's a straightforward way I can make the error cases also respect `_return_url`.